wiki/configuration.md

```diff
@@ -0,0 +1,17 @@
+<div id="main_content_wrap" class="outer">
```

Comment:
Same comment here regarding the html tags. Remove them from all the files, since they're not needed.

Reply:
removed html tags from all files